--- lib/node-gd.js	2020-09-11 21:24:59.945903226 +0200
+++ lib/node-gd-new.js	2020-09-11 21:44:00.075234552 +0200
@@ -85,10 +85,11 @@
     return;
   }
 
-  if (!`save${format}`) Object.defineProperty(
-    bindings.Image.prototype, `save${format}`, {
-    value: saveFormatFn(format)
-  });
+  if (!bindings.Image.prototype[`save${format}`]){
+    Object.defineProperty(bindings.Image.prototype, `save${format}`, {
+      value: saveFormatFn(format)
+    });
+  }
 });
 
 /**
